changelog.txt

```diff
@@ -24,8 +27,7 @@
 * Fix - Empty file input to allow the user to select the same file again if there's an error.
 * Fix - Enable card readers branding section.
 * Fix - Enable WooCommerce Blocks checkout to hide option to save payment methods for a non-reusable payment method.
-* Fix - Fix - Using any other payment methods apart from WooCommerce Payments in "Pay for order".
-form triggers validation errors when UPE checkout is enabled.
+* Fix - Using any other payment methods apart from WooCommerce Payments in "Pay for order" form triggers validation errors when UPE checkout is enabled.
```

Comment:
Cherry-picking ca0aa7a9160647e3612bffc8b9e0f935bea3acd1 because malformatted changelog blocks the `changelog write` command that adds changelog entries.